share/github-backup-utils/ghe-backup-settings

```diff
@@ -26,16 +26,51 @@ echo "* Transferring license data ..." 1>&3
 ghe-ssh "$host" -- "sudo cat '$GHE_REMOTE_LICENSE_FILE'" > enterprise.ghl
 
 # Function to backup a secret setting to a file.
-#   backup-secret <description> <file-name> <setting-name>
+#   backup-secret <description> <file-name> <setting-name> 
```

Comment:
```suggestion
#   backup-secret <description> <file-name> <setting-name>
```

Reply:
I added in the new switch as well, and made sure not leave a trailing space.